packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```
@@ -626,6 +627,173 @@
         }
       },
       "required": ["type", "image"]
+    },
+    "guidancePanel": {
+      "type": "object",
+      "description": "The guidance panel specification for the page.",
+      "additionalProperties": false,
+      "required": ["title"],
+      "properties": {
+        "title": {
+          "type": "string",
+          "description": "Guidance panel title",
+          "doNotSuggest": false
+        },
+        "backgroundImage": {
+          "type": "object",
+          "description": "Image to display in the background of this guidance panel. It should be a horizontal image and it will be fixed at the bottom of the guidance panel.",
```

In the guidance panel case, the image should actually be a vertical image, since, unlike the regular page backgroundImage, the guidance panel will normally be taller than wide.
```suggestion
          "description": "Image to display in the background of this guidance panel. It should be a vertical image and it will be fixed at the bottom of the guidance panel.",
```